solutions/4.js

```diff
@@ -0,0 +1,8 @@
+const solution = (str, num, i = 0, newStr = '') => {
+  if(i == num){
```

Comment:
side note (you don't have to fix this). If you only have 1 thing to do in if statement, you can simply write everything in 1 line:
`if (i === num) return newStr;`